### ISSUE TITLE
Invocation of Log Viewer gives no feedback if logging level is set to disabled and there is no log to view

### ISSUE THREAD

**Issue by britechguy** [nvaccess/nvda#14998]
### Steps to reproduce:
1. Set logging level in NVDA to "disabled" in NVDA General Settings.

2. Hit NVDA + F1 to invoke log viewer.  Nothing happens, and no message is issued that logging is currently disabled and there is no log available to view.

### Actual behavior:
Described above.

### Expected behavior:
When NVDA + F1 is issued to invoke the Log Viewer, but logging is disabled and no log exists to view, the user should be given a notification that this is the case.  Having absolutely nothing happen is confusing as logging level can be changed and forgotten, or accidentally changed.  When logging is disabled, the end user should be told this if they attempt to open the NVDA log in Log Viewer.

### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.1
#### Windows version:
Win11 Version 22H2
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by CyrilleB79** [comment]
NVAccess staff, I think that the use case of log disabled should be clarified since we have the following situations that do not match each other when log is disabled:
1. NVDA+F1 does not open the log viewer and nothing is reported:
  * either the log should be opened with dev information on the object
  * or it should be reported to the user that dev info cannot be logged since log is deactivated. In this case:
    * either the log is opened anyway; it may have content if the log level has been modified during the current NVDA session
    * or the log viewer is not opened at all
2. Opening the log viewer via NVDA menu is still possible. The log may have content if the log level has been modified during the current NVDA session.

I would recommend the following:
* Setting log level to Disabled should prevent from logging any message but not to open the log viewer.
* As a consequence NVDA+F1 should allow to open the log viewer in any case, i.e. also with log level = disabled.
* If log is disabled, it should not be allowed to log anything thus dev info cannot be logged either. Maybe a message in the log viewer should indicate that no dev info can be logged due to log level set to disabled; but this message should not appear in the logfile nor when reopening or updating the log viewer later.

What do you think?

**Comment by seanbudd** [comment]
@CyrilleB79 that plan looks generally good

>If log is disabled, it should not be allowed to log anything thus dev info cannot be logged either. Maybe a message in the log viewer should indicate that no dev info can be logged due to log level set to disabled; but this message should not appear in the logfile nor when reopening or updating the log viewer later.

Instead of this, I think we should always load the log viewer with the contents of "Logging is disabled". instead of a temporary and specific message about dev info.
